webpack.config.publish.js

```diff
@@ -11,33 +13,19 @@ module.exports = baseConfig({
     filename: 'index.js',
     libraryTarget: 'umd'
   },
-  externals: {
-    react: {
-      root: 'React',
-      commonjs2: 'react',
-      commonjs: 'react',
-      amd: 'react',
-      umd: 'react',
-    },
-    'react-dom': {
-      root: 'ReactDOM',
-      commonjs2: 'react-dom',
-      commonjs: 'react-dom',
-      amd: 'react-dom',
-      umd: 'react-dom',
-    },
-    lodash: 'lodash',
-    classnames: 'classnames'
-  },
+  externals: [ nodeExternals() ],
   plugins: [
     new OptimizeCSSAssetsPlugin(),
-    new webpack.optimize.UglifyJsPlugin({
-      minimize: false,
-      mangle: false,
-      compress: {
-        warnings: false
-      },
-      sourceMap: true
+    new UglifyJsPlugin({
+      sourceMap: false,
+      uglifyOptions: {
+        minimize: true,
```

Comment:
FWIW I think the default value is to minimize, but it is fine to be explicit here

Reply:
lol, @PepperTeasdale I'm pretty sure this isn't even a real prop - I left it there just in case it is, but I don't think it does anything.